### ISSUE TITLE
Question about AdoptOpenJDK security patches strategy

### ISSUE THREAD

**Issue by SzentnerTsIT** [AdoptOpenJDK/openjdk-build#619]
Dear OpenJDK,

we see, that currently AdoptOpenJDK 8 is on security patch 172, but the most up-to-date state is 181. We saw as well, that 181 will come out soon. We have two questions:
- When is it realistic, that 181 will be available ?
- Can it be expected for the future, that the Adopt distributions will have the most up-to-date security patches or will Adopt usually be behind?

Thanks and kind regards,
Tamás

**Comment by karianna** [comment]
Hi Tamas,

We'll try to get a 181 release out this week or next at the latest.  We
anticipate being only a day or 2 behind for future releases

Cheers,
Martijn

**Comment by SzentnerTsIT** [comment]
Hallo Martijn,

great, thanks!

Kind regards,
Tamás

**Comment by ashishshraff** [comment]
Hi Team,

Do you release Security patches as part of new released version or is there a separate forum to get that information?

Regards,
Ashish

**Comment by karianna** [comment]
Yes security patches are released as part of the release.  see the release_notes page at adoptopenjdk.net
